interface/src/pages/index.tsx

```diff
@@ -0,0 +1,68 @@
+import Link from "next/link";
+import { Head } from "@/components/layout/Head";
+import { SITE_DESCRIPTION } from "@/lib/constants";
+
+const Home = () => {
+  // eslint-disable-next-line no-console
+  console.log(`
+:'######::'########::'########::::'###::::'########:'########:'##::::'##:
+'##... ##: ##.... ##: ##.....::::'## ##:::... ##..:: ##.....::. ##::'##::
+ ##:::..:: ##:::: ##: ##::::::::'##:. ##::::: ##:::: ##::::::::. ##'##:::
+ ##::::::: ########:: ######:::'##:::. ##:::: ##:::: ######:::::. ###::::
+ ##::::::: ##.. ##::: ##...:::: #########:::: ##:::: ##...:::::: ## ##:::
+ ##::: ##: ##::. ##:: ##::::::: ##.... ##:::: ##:::: ##:::::::: ##:. ##::
+. ######:: ##:::. ##: ########: ##:::: ##:::: ##:::: ########: ##:::. ##:
+:......:::..:::::..::........::..:::::..:::::..:::::........::..:::::..::
+`);
+  const cards = [
+    {
+      id: 1,
+      href: "/deployments",
+      title: "Deployments",
+      subtitle: "Deployed on 60+ chains",
```

Comment:
You may have noticed this part is a constant, I just periodically update it and re-deploy. This is mainly because I didn't want to require an unnecessary http request to load the home page